### PR TITLE
Revert "Issue #1087 redis fix"

### PR DESCRIPTION
**Pull request by auvipy**
Reverts celery/kombu#1089

### CODE REVIEW

**Comment by thedrow** [comment]
Why are we reverting this?

**Comment by auvipy** [comment]
this might have introduced a regression

**Comment by thedrow** [comment]
How exactly?
This brings support for py-redis 2.x which we don't want.

**Comment by littlebtc** [comment]
See #1101

**Comment by auvipy** [comment]
I will handle pyredis 3.x in a separate PR. this doesn't seem to add pyresis 2.x though

File: kombu/transport/redis.py [hunk 1]
```diff
@@ -147,8 +146,12 @@ def __init__(self, *args, **kwargs):
     def append(self, message, delivery_tag):
         delivery = message.delivery_info
         EX, RK = delivery['exchange'], delivery['routing_key']
-        # Redis-py changed the format of zadd args in v3.0.0 to be like this
-        zadd_args = [{delivery_tag: time()}]
+        # TODO: Remove this once we soley on Redis-py 3.0.0+
+        if redis.VERSION[0] >= 3:
```
Comment (thedrow) [inline kombu/transport/redis.py]: Yes it does...

Comment (auvipy) [reply kombu/transport/redis.py]: I will handle this in a pr after merging this

**Comment by matteius** [comment]
@auvipy  This is rough news to see that my work was reverted -- I had spent quite a few long nights and hours on #1089 which you reverted.  All of the work to get the module to use fakeredis for the tests as well as fixing the other reports of #1087 and #1081 ... I don't understand how we let a different bug report cause us to revert back to a point that was known to have issues.   I want to help but when my work is just flat ripped out after it was accepted back in August makes it hard.

**Comment by matteius** [comment]
I honestly think we are making a mistake, we did a bunch of work on Kombu to make it better with respect to some bug reports, there is a new report, we should have continued improving the code from that point, but then a number of revisions have been made to turn it it back closer into a much older release of Kombu -- without much discussion. 
image

I am going to diff where trunk is now to prior release or two, and also compare to where it was when I was working on it pre-October with respect to Redis and what the report of #1101 was, because it would have been nice to fix #1101 under the new framework that was also using fakeredis client in the tests.

**Pull request merged**